presto-main/src/main/java/com/facebook/presto/execution/ClusterSizeMonitor.java

```diff
@@ -88,7 +95,9 @@ public ClusterSizeMonitor(
             int workerMinCountActive,
             Duration executionMaxWait,
             int coordinatorMinCount,
-            Duration coordinatorMaxWait)
+            int coordinatorMinCountActive,
```

Comment:
It looks like we are checking current active coordinator >= coordinatorMinCount in waitForMinimumCoordinators method. What is the difference between this newly introduced field (coordinatorMinCountActive) and coordinatorMinCount?

Reply:
`coordinatorMinCount` is no longer needed and some other code, so removing it.